styleguide/source/assets/scss/01-atoms/_section-title.scss

```diff
@@ -3,9 +3,12 @@
   @extend %ama__h2--homepage;
 
   @include breakpoint($bp-small max-width){
```

Comment:
We probably should stay away from max-widths. Max widths are not responsive first

Reply:
This isn't a change that was made with this PR and as such seems like it is outside the scope of the PR.